src/sentry/search/utils.py

```diff
@@ -145,6 +147,26 @@ def get_date_params(value, from_field, to_field):
         })
     return result
 
+reserved_tag_names = frozenset([
```

Comment:
Another nitpit, this occupies a lot of lines since it's one word per line. I'd just use normal word-wrap here. Something like: https://github.com/getsentry/sentry/blob/master/src/sentry/constants.py#L76-L86

But not a big deal, just my personal preference in this case.